lm_eval/tasks/multimedqa/_multimedqa.yaml

```diff
@@ -0,0 +1,11 @@
+group: multimedqa
+task:
+  - pubmedqa
+  - medmcqa
+  - medqa_4options
+  - mmlu_anatomy
+  - mmlu_clinical_knowledge
+  - mmlu_college_medicine
+  - mmlu_medical_genetics
+  - mmlu_professional_medicine
+  - mmlu_college_biology
```

Comment:
This could be moved to `lm_eval/tasks/benchmark/`. It's a put benchmark configs.